Streetcode/Streetcode.XUnitTest/MediatRTests/StreetCode/Terms/CreateTermHandlerTests.cs

```diff
@@ -0,0 +1,99 @@
+﻿using AutoMapper;
+using MediatR;
+using Moq;
+using Streetcode.BLL.DTO.Streetcode.TextContent;
+using Streetcode.BLL.MediatR.Streetcode.Term.Create;
+using Streetcode.DAL.Entities.Streetcode.TextContent;
+using Streetcode.DAL.Repositories.Interfaces.Base;
+
+using Xunit;
+
+namespace Streetcode.XUnitTest.MediatRTests.StreetCode.Terms
+{
+	public class CreateTermHandlerTests
+	{
+		private Mock<IRepositoryWrapper> _mockRepository;
+		private Mock<IMapper> _mockMapper;
+
+		public CreateTermHandlerTests() 
+		{
+			_mockMapper = new();
+			_mockRepository = new();
+		}
+
+		[Fact]
+		public async Task ShouldReturnSuccessfully_TypeIsCorrect() {
+			//Arrange
+			_mockRepository.Setup(x => x.TermRepository.Create(GetTerm()));
+			_mockRepository.Setup(x => x.SaveChangesAsync()).ReturnsAsync(1);
+
+			_mockMapper.Setup(x => x.Map<Term>(It.IsAny<TermDTO>()))
+				.Returns(GetTerm());
+
+			var handler = new CreateTermHandler(_mockMapper.Object, _mockRepository.Object);
+
+			//Act
+			var result = await handler.Handle(new CreateTermCommand(GetTermDTO()), CancellationToken.None);
+
+			//Assert
+			Assert.IsType<Unit>(result.Value);
+
+		}
+		
+		[Fact]
+		public async Task ShouldReturnSuccessfully_WhenTermAdded() {
+			//Arrange
+			_mockRepository.Setup(x => x.TermRepository.Create(GetTerm()));
+			_mockRepository.Setup(x=>x.SaveChangesAsync()).ReturnsAsync(1);
+
+			_mockMapper.Setup(x => x.Map<Term>(It.IsAny<TermDTO>()))
+				.Returns(GetTerm());
+
+			var hendler = new CreateTermHandler(_mockMapper.Object, _mockRepository.Object);
+
+			//Act
+			var result = await hendler.Handle(new CreateTermCommand(GetTermDTO()), CancellationToken.None);
+
+			//Assert
+			Assert.True(result.IsSuccess);
+		}
+		
+		[Fact]
+		public async Task ShouldThrowException_WhenTryToAddNull() {
+			//Arrange
+			_mockRepository.Setup(x => x.TermRepository.Create(GetTerm()));
+			_mockRepository.Setup(x => x.SaveChangesAsync()).ReturnsAsync(1);
+
+			_mockMapper.Setup(x => x.Map<Term>(It.IsAny<TermDTO>()))
+				.Returns(GetNotExistingTerm()!);
+
+			var expectedError = "Cannot convert null to Term";
+			var hendler = new CreateTermHandler(_mockMapper.Object, _mockRepository.Object);
+
+			//Act
+			var result = await hendler.Handle(new CreateTermCommand(GetNotExistingTermDTO()!), CancellationToken.None);
+
+			//Assert
+			Assert.Multiple(
+				()=>Assert.Equal(expectedError, result.Errors.First().Message),
+				()=>Assert.False(result.IsSuccess)
+				);
+		}
+
+		private static Term GetTerm() {
+			return new();
```

Comment:
you can either reduce code complexity by using this: 
private static Term GetTerm() => new(); 
or move curly brace to the next line.